lexicons/app/bsky/feed/postgate.json

```diff
@@ -0,0 +1,32 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.feed.postgate",
+  "defs": {
+    "main": {
+      "type": "record",
+      "key": "tid",
+      "description": "Record defining interaction rules for a post. The record key (rkey) of the postgate record must match the record key of the post, and that record must be in the same repository.",
+      "record": {
+        "type": "object",
+        "required": ["post", "createdAt"],
+        "properties": {
+          "createdAt": { "type": "string", "format": "datetime" },
+          "post": {
+            "type": "string",
+            "format": "at-uri",
+            "description": "Reference (AT-URI) to the post record."
+          },
+          "detachedQuotes": {
+            "type": "array",
+            "maxLength": 50,
+            "items": {
+              "type": "string",
+              "format": "at-uri"
+            },
+            "description": "List of detached quote post URIs."
+          }
```

Comment:
```suggestion
          },
          "rules": {
            "type": "array",
            "maxLength": 5,
            "items": {
              "type": "union",
              "refs": ["#noQP", "#followingRule"]
            }
          }
```

Reply:
Thanks for this. I think we probably want to keep parity with threadgates in terms of naming and features. I'll push something up and we can discuss in Slack 👍